kdenlive/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "2.1"
```

Comment:
For all of our apps, we've tried to standardise on version `3.7`, could you change this please?

Reply:
done changes for this